configure.ac

```diff
@@ -62,7 +62,7 @@ MONO_VERSION_BUILD=`echo $VERSION | cut -d . -f 3`
 # This line is parsed by tools besides autoconf, such as msvc/mono.winconfig.targets.
 # It should remain in the format they expect.
 #
-MONO_CORLIB_VERSION=8E172F4D-82A3-4F55-8D7C-C8E8E0FA58AE
+MONO_CORLIB_VERSION=8E172F4D-82A3-4F55-8D7C-C8E8E0FA58AF
```

Comment:
No need for a corlib bump.

Reply:
Oh, alright, I will revert it when I get back to my computer.

Reply:
This confuses me. While a newer same-guid BCL will continue to work with old and new runtime, there exist older same-guid BCL that will not work with this newer runtime.